### ISSUE TITLE
pythonPackages.wxPython: doesn't build on master

### ISSUE THREAD

**Issue by pbogdan** [NixOS/nixpkgs#56569]
## Issue description

The package doesn't build.

### Steps to reproduce

Run

```
$ nix-build -A pythonPackages.wxPython
```

and observe build failure.

## Technical details

nixpkgs @ 3944fa41a05f5334ec1f9d5405bfab863b9ba6f2

**Comment by mtrsk** [comment]
wxPython-3.0.2.0 is also falling during build of the Nvidia driver on unstable (`26d8a8c0eb2a88d55694249b099cdd2c89b2b06d`) . The culprit seems to be wxwidgets:

```
In file included from /nix/store/h6f31s0hlf8kp10ji4pgfxblh59cy97v-wxwidgets-3.0.4/include/wx/defs.h:27:0,
                 from /nix/store/h6f31s0hlf8kp10ji4pgfxblh59cy97v-wxwidgets-3.0.4/include/wx/wx.h:14,
                 from include/wx/wxPython/wxPython_int.h:35,
                 from src/helpers.cpp:17:
/nix/store/h6f31s0hlf8kp10ji4pgfxblh59cy97v-wxwidgets-3.0.4/include/wx/platform.h:183:10: fatal error: wx/setup.h: No such file or directory
 #include "wx/setup.h"
          ^~~~~~~~~~~~
compilation terminated.
error: command 'gcc' failed with exit status 1
```

**Comment by FRidh** [comment]
This is due to b4acd9772975d549f491d48debb679cf4ec78133.

**Comment by FRidh** [comment]
Fixed in 1285d739d8233ad7804c6eafb2f5766e021e973d and cherry-picked to staging-19.03
